### DISCUSSION TITLE
Setting recorder up

### DISCUSSION THREAD

**Discussion by Chronos16** [lauwarm/docker-streamlink-recorder#49] [General]
Hey could anyone please help me out setting this tool up?

**Comment by lauwarm** [comment]
Hey @Chronos16,

Sorry for the late reply. What exactly do you need help with? 

Generally speaking you need to:
1. Install [Docker](https://www.docker.com/products/docker-desktop/) on your System.
2. Maybe run the _[Hello World](https://hub.docker.com/_/hello-world)_ example to verify docker is working.
3. Depending on your OS you need to find your UserID and GroupID.
4. From this point onwards you should be able to copy the example and give it a go.

**Reply by Chronos16** [reply]
hey!
no worries
i found someone that guided me a bit through it as im not that experienced with it

But rather another topic. Is it possible to also include the twitch chat in the recording?

**Reply by lauwarm** [reply]
From the official streamlink website:

Streamlink is a [command-line utility](https://streamlink.github.io/cli.html#command-line-interface) which pipes video streams from various services into a video player, such as [VLC](https://www.videolan.org/).

So no, it cannot record the chat.

**Reply by XGxF3** [reply]
There is a [chat-downloader](https://github.com/xenova/chat-downloader) that might work well with your container.

**Reply by lauwarm** [reply]
Hey @XGxF3,

Thanks for the information. I did a short test and it seems to save the chat. Do you know if chat-downloader stops itself when a livestream stops?

Anyways, I made a Docker Image for this App. It can be found [here](https://github.com/lauwarm/docker-chat-downloader) if you want to use it.

**Reply by XGxF3** [reply]
Thanks for making the docker container. No the chat-downloader does not stop when the stream ends.
